cmd/fluxd/main.go

```diff
@@ -38,11 +38,12 @@ import (
 var version string
 
 const (
-	defaultRemoteConnections = 125 // Chosen performance tests on sock-shop. Unable to get higher performance than this.
-	// Memcache: There is only one thread to accept new client connections. If you are cycling connections very quickly, you can overwhelm the thread. Use persistent connections or UDP in this case.
-	// See https://github.com/memcached/memcached/wiki/Performance
-	// Hence, keep the number of new connections low.
-	defaultMemcacheConnections = 10
+	// The number of connections chosen for memcache and remote GETs must match.
+	// If they do not match then the remote either creates too many memcache connections
+	// or not enough to be efficient.
```

Comment:
I think this comment, in light of what `MaxIdleConns` actually does, is a bit strongly worded! It's more like a rule of thumb ...